src/components/Alert/AlertContainer.js

```diff
@@ -0,0 +1,107 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import Alert from 'react-alert';
+
+import { updateThershold, updateInterval } from './../../store/actions';
+import {
+    selectCurrentInterval,
+    selectUptimes,
+    selectCurrentTimerCount
+} from './../../store/selectors';
+
+import { currentAverage } from '../../utils/intervals';
+import {
+    TIME_LIMIT,
+    UPTIME_FECTH_INTERVAL,
+    AVERAGE_UPTIME_INTERVAL
+} from '../../utils/constants';
+import { ALERT_OPTIONS } from './alertOptions';
+import { errorMessage, sucessMessage } from './alertMessages';
+
+class AlertContainer extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            errorHasOccured: false
+        };
+    }
+
+    componentWillReceiveProps(nextProps) {
+        const { currentTimerCount } = this.props;
+
+        if (
+            nextProps.currentTimerCount !== currentTimerCount &&
```

Comment:
Could you maybe give these conditions a more semantic name ?

It's hard when first looking at it what the condition is meant to be doing. 

e.g 
```javascript
const meaningfulName = (nextProps.currentTimerCount !== currentTimerCount) &&  (currentTimerCount % UPTIME_FECTH_INTERVAL === 0);
```